### PR TITLE
{phys}[goolf/1.7.20] Atomsk vb0.9.10

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/a/Atomsk/Atomsk-b0.9.10-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+name = 'Atomsk'
+version = 'b0.9.10'
+
+homepage = 'http://atomsk.univ-lille1.fr/index.php'
+description = "Atomsk is an Open Source command-line program dedicated to the creation, manipulation, and conversion of atomic systems"
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['http://atomsk.univ-lille1.fr/code/']
+sources = ['%(namelower)s_%(version)s.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/a/Atomsk/Atomsk-b0.9.10-goolf-1.7.20.eb]: @pescobar On top of the style issues reported, please add a SHA256 checksum

File: easybuild/easyconfigs/a/Atomsk/Atomsk-b0.9.10-goolf-1.7.20.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+start_dir = 'src'
+
+buildopts = " atomsk LAPACK=${LIBLAPACK}"
+
+files_to_copy = [(['atomsk'], 'bin'), '../*']
```
Comment (boegel) [inline easybuild/easyconfigs/a/Atomsk/Atomsk-b0.9.10-goolf-1.7.20.eb]: What else are you copying with `../*`? Please include some of it in `sanity_check_paths`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2079.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6b3ab2996a85e68b835944ed3464f480 for a full test report.

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): uses a deprecated toolchain.
For more information on deprecated toolchains, see https://easybuild.readthedocs.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains.
Please consider reopening or submitting a new PR using a supported toolchain.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**